internal/api/front/perses/src/app/project/dashboard/dashboard-details/dashboard-details.component.ts

```diff
@@ -25,29 +30,74 @@ import { DashboardModel } from '../dashboard.model';
   styleUrls: ['./dashboard-details.component.scss']
 })
 export class DashboardDetailsComponent implements OnInit {
+  private readonly paramName = 'dashboard';
+  name = '';
   isLoading = false;
-  currentProject = '';
   dashboard: DashboardModel = {} as DashboardModel;
+  ngxData = new Map<string, NgxLineChartModel[]>();
+  currentProject = '';
+
+  // TEMP static options data for graph display
+  view: [number, number] = [600, 400];
+  showXAxis = true;
+  showYAxis = true;
+  gradient = false;
+  showLegend = true;
+  showXAxisLabel = true;
+  xAxisLabel = 'Time';
+  showYAxisLabel = true;
+  yAxisLabel = 'Value';
+  timeline = true;
+  colorScheme = {
+    domain: ['#9370DB', '#87CEFA', '#FA8072', '#FF7F50', '#90EE90', '#9370DB']
+  };
 
-  constructor(private dashboardService: DashboardService,
+  constructor(private service: DashboardService,
+              private feedService: DashboardFeedService,
               private readonly toastService: ToastService,
-              private readonly projectService: ProjectService) {
+              private readonly projectService: ProjectService,
+              private route: ActivatedRoute) {
```

Comment:
mmm I think you would also need the service for the theme. To be able to change the theme of the chart. (But it could done in the dedicated component (for the display of the chart) I suggested somewhere in my comments)

Reply:
added to dashboard-sections component (but not used yet)